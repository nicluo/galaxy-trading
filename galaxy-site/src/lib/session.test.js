import React, { Component } from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import withSession from './session';

describe('session', () => {
  let WrapperComponent;

  beforeEach(() => {
    class MockApp extends Component {
      render() {
        return (<div>App</div>);
      }
    }

    WrapperComponent = withSession(MockApp);
  });

  test('withSession should render without crashing', () => {
    const wrapper = shallow(<WrapperComponent/>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
