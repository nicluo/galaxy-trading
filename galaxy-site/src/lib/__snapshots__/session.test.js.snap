// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`session withSession should render without crashing 1`] = `
<MockApp
  offline={false}
  parserId={null}
  sessionId={null}
/>
`;
